### DISCUSSION TITLE
Referalimdan gelen kayitlarin ibaxlari hesabimda gozukmuyor

### DISCUSSION THREAD

**Discussion by Sarius1996** [IBAX-io/go-ibax#794] [Soru Cevap - Türkçe]
Merhaba referalimla bir cok arkadasim kayd oldu ve gorevleri tamamladi lakin bende referal kisminda cikmiyolar ve puanlari gozukmuyor

**Comment by ghost** [comment]
Bilen varsa plsssss

**Comment by Yusufknk** [comment]
Her iki tarafın da tüm görevleri yapmaları ve airdrop sonuna kadar sosyal medya kanallarını takipten çıkarmamaları  gerekir. Bu şartları sağlayan kullanıcıların ödülleri yansıyacaktır.

**Reply by Brown2222** [reply]
Ben de de aynı sorun var
